tests/lcs/core/files/bom.livecodescript

```diff
@@ -0,0 +1,38 @@
+﻿script "CoreFilesBOM"
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+constant kStackName = "utf8bom"
+constant kTestFile = "utf8bom.livecodescript"
+
+on TestLoadBomScriptOnlyStack
+   local tUrlBinfilePath
+   
+   put quote & "binfile:" & testFilesFolder() & slash & \
+      kTestFile & quote into tUrlBinfilePath
+   
+   open stack url tUrlBinfilePath
+   
+   // Technically this is testing the opening of a stack that
+   // is coming from a chunk.  Most likely source would be URL
+   // "binfile:" and "http(s):", but others could be possible.
```

Comment:
The file should be good.  It does report the size as 62 bytes on GitHub which matches my local system copy.  In DP8 I can open it using binfile only with the char 4 to -1 trick.  Now that I'm thinking about it though, maybe I just need to create a variable with the BOM and use that.